### DISCUSSION TITLE
Iranian VPS <-> non-Iranian VPS

### DISCUSSION THREAD

**Discussion by mfathi91** [WeAreMahsaAmini/Core#230] [Ideas]
Hello guys, thanks for the great initiative.

I personally believe that the best way that we—as tech guys—can help the people of Iran is to give them back their internet. Of course it might be helpful to raise the awareness of non-Iranians about what is happening in Iran, but the real changes will happen only by the people INSIDE Iran.

As we know the trivial VPNs don't work these days, so in order to make things work we need to have two VPSs, one inside Iran, and one outside (e.g. in Germany). Purchasing a VPS in Europe is fairly easy for us, but the tricky part is to buy the Iranian VPS (e.g. Parspack).

Long story short, let this discussion be a place to:
* share the ways that you guys buy your Iranian VPSs
* connect the guys who have non-Iranina VPS with the guys who have Iranian VPS, so that they can work together to configure e.g. SSH tunneling to make a robust VPN/proxy

**Comment by ghost** [comment]
This sounds like a great idea. And I'm not an expert, just trying to get an understanding of what it will take: What is needed in terms of infrastructure inside Iran to make this a reality? I think if we can break this down into steps, it might help to make it achieveable.

**Comment by IranZop** [comment]
I can help with the non-Iranian VPS side. Happy to give it a go If we find the Iranian counterpart.

**Comment by theStratus** [comment]
من از ابر اوران خوشم نمیاد. ولی خب اونجا حساب کاربری دارم. برای تست میتونم در اختیار بزارم ولی برای پروداکشن نیاز به یک حساب کاربری با هویت دیگه ای دارم.

**Reply by mfathi91** [reply]
اگه یه VPS سبک داری و اوکی هستی که من برای کانفیگ کردنش ازش استفاده کنم، میتونی آی پی و پسورد رووت رو برام بفرستی. من مشخصات وی پی انی که میارم بالا رو برات میفرستم و با بقیه هم share میکنم (نه به صورت عمومی).

**Reply by theStratus** [reply]
Emailed

**Reply by mfathi91** [reply]
هنوز ایمیلی نگرفتم.

**Comment by theStratus** [comment]


**Reply by mfathi91** [reply]
You're right. It was in the spam folder. I'll configure it, and will let you know tomorrow. Thanks

**Reply by mfathi91** [reply]
@norits021 check your mail...

**Comment by itsmohsenjalali** [comment]
سلام من چند روزی هست درگیر راه اندازی وی پی ان بودم و تقریبا چندتایی هم امتحان کردم
که اینجا مینویسم:
1- wireguard
2- shadowsocks/ v2ray
3- openvpn
4- ssh tunnel
مورد اول و دوم رو خودم امتحان کردم و مشکل اینه که توی ایران مستقیم بستن و یا زود شناسایی میشه و بسته میشه مورد سوم رو یکی از دوستام داشت که از ۲ روز پیش اونم به مشکل خورد و مورد ۴ تنها راهکاری بود که تا الان خیلی خوب کار داده و هم مستقیم میشه ازش استفاده کرد هم میشه از طریق پروکسی کردن یک سرور داخلی از استفاده کرد که فعلا بسته نشده(چون انگار تو لایه دیتاسنتر شروع میکنن مونیتور کردن و روی پروتکل و پورت و ورودی و خروجی قرینه)
اهان یه نکته دیگه فندق یه سرویس دهنده ی فضای ابریه که پلن رایگان هم داره چک نکردم ولی میتونه کمک کننده باشه
و یه نکته اینه برای ssh بزارید روی پورت هایی مثل 443 اینطوری کمتر احتمال داره ببندن

**Comment by mfathi91** [comment]
https://github.com/InternetForIran/InternetForIran
